compendium/tools/SCRCodeGen/ManifestParserImpl.cpp

```diff
@@ -70,13 +70,14 @@ ManifestParserImplV1::ParseAndGetComponentInfos(Json::Value const& scr) const
             componentInfo.configurationPolicy
                 = JsonValueValidator(jsonComponent, "configuration-policy", policyChoices).GetString();
         }
+        std::unordered_map<std::string, std::string> duplicatePids;
```

Comment:
move the construction closer to where its used.

Reply:
This is used in the if statement that follows this declaration but I can't move it inside the if statement because it is used later on line 108. It has to be defined outside the if block

Reply:
I missed that. ok, nevermind then.